mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go

```diff
@@ -0,0 +1,124 @@
+package securitycenterv2_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google"
```

Comment:
There seems to be an issue importing this. I don't think we generally do import it. See other comments.

Reply:
addressed